docs/runtimes/application.ipynb

```diff
@@ -92,6 +92,74 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    # for Basic authentication mode, credentials are also required.
</pre><p> </p><p>some inconsistencies here with capital letters not clear why</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>